### ISSUE TITLE
Dublicated setNeutralWhite causes in reduced brightness.

### ISSUE THREAD

**Issue by DivineThreepwood** [openbase/bco.dal#67]
If the "setNeutralWhite" is triggered more than once on a location remote the room is reducing its brightness.

* Please try to reproduce and fix this bug.
* Patch 1.4 and master

**Comment by pLeminoq** [comment]
I did some changes in commit adb568e71f3ceb9f3d8b8ec65b43588459cc5e50. But I only tested it in hardware simulation mode. The following issues existed:

* setNeutralWhite did not use the action pipeline but was a direct rpc call
* Locations did not trigger setNeutralWhite on every unit separately but used the default implementation which caused that the default color would be used even if a different one has been defined in the registry for a certain device class
* ColorableLightRemote now converts colors to HSB if they are given in RGB before applying them

The two last ones are the important ones because the default neutral white is defined in RGB. Thus triggering setNeutralWhite on a location should not have worked correctly at all.
